src/ccextractor.c

```diff
@@ -202,8 +202,7 @@ int api_start(struct ccx_s_options api_options)
 #ifdef ENABLE_FFMPEG
                 case CCX_SM_FFMPEG:
 #endif
-                if (!api_options.use_gop_as_pts) // If !0 then the user selected something
-                    api_options.use_gop_as_pts = 0;
+                api_options.use_gop_as_pts = 0;
```

Comment:
I'm not so sure if this is the correct logic.

if the use_gop_as_pts is set to 1, it'll skip the statement under old conditions. Now it'd be set to 0.
It seems more logic to me to just delete the two lines.

@cfsmp3 What do you think?

Reply:
And since this has been in since the original commit of the repo, we definitely need @cfsmp3 to tell something about it :+1:

Reply:
I wrote this more than 1 month ago and just realised it doesn't make sense at all.

if it is 0 then !0 == 1

so the variable is set to 0

And if it is 1 (or anything else) nothing happens.

Reply:
I removed the lines.

Also my previous commit was totally wrong as it would zero the variable when it is one which it didn't before